portfolio/src/main/webapp/script.js

```diff
@@ -42,4 +42,28 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
+}
+
+async function loadTasks() {
+    let list = await fetch('/list-tasks').then(response => response.json());
+    for(c in list){ 
+        let reviewsElem = document.getElementById("reviews");
+        let commentElem = document.createElement("DIV");
+        commentElem.appendChild(document.createTextNode(list[c].email+": "+list[c].description));
+    }
+    document.getElementById("showCQC").onclick = null;
+}
+async function loginStatus() {
+    let status = await fetch('/status').then(response => response.json());
+    console.log(status);
+    if (status=='Currently logged in as Guest.'){
+        document.getElementById("cqcForm").style.display="none";
+    }else{
+        document.getElementById("cqcForm").style.display="block";
+    }
+}
+
+async function getEmail(){
+    let email = await fetch('/user').then(response => response.json());
```

Comment:
Please do not mix promises and awaits. Also, it's not clear what this function is trying to do? Do you want to return something here?